src/components/Profile/ProfileInfoCard/index.js

```diff
@@ -102,9 +100,11 @@ const ProfileInfoCard = () => {
 
   const uploadImage = (file) => {
     setImageUploading(true);
-    uploadProfileImage(file, profileData.handle)(firebase, dispatch).then(() => {
-      setImageUploading(false);
-    });
+    uploadProfileImage(file, profileData.handle)(firebase, dispatch).then(
+      () => {
+        setImageUploading(false);
+      }
+    );
```

Comment:
Better to revert into original.

Reply:
This was the result of running ```prettier``` over the file. I did not make changes to this code. Should I revert it?

Reply:
@Niweera Should i open this as an issue for the project? 
There must be ```.prettierrc``` config file at the root of the project to avoid different linting patterns.

Reply:
Yes. You can create an issue.